boltons/iterutils.py

```diff
@@ -374,26 +374,50 @@ def chunked_iter(src, size, **kw):
     return
 
 
-def chunked_filter(iterable, predicate, chunk_size):
-    """A version of :func:`filter` which will call predicate with a chunk of the iterable.
+def chunked_filter(iterable, predicate, size):
```

Comment:
oops, based on the docstring, I think you meant to rename `predicate` here to `key`.

Reply:
not sure, see previous comments